tests/test_execution.py

```diff
@@ -4,6 +4,9 @@
 
 import substratest as sbt
 
+status_done = sbt.client.assets.TupleStatus.done.name
+status_failed = sbt.client.assets.TupleStatus.failed.name
```

Comment:
Why creating global variables here? We could directly import the desired module. 

To avoid having a too long path we could import the assets module:
`from substratest import assets`